homeassistant/components/sonos/media_player.py

```diff
@@ -186,7 +187,12 @@ async def async_service_handle(service_call: ServiceCall) -> None:
         {vol.Optional(ATTR_QUEUE_POSITION): cv.positive_int},
         "remove_from_queue",
     )
-
+    
+    platform.async_register_entity_service(
+        SERVICE_PARTYMODE,
+        {},
+        "party_mode_services",
```

Comment:
To be consistent with the other services registered here, this string should be "partymode"